FlowCryptUI/Cell Nodes/ThreadDetailWebNode.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ThreadDetailWebNode.swift
+//  FlowCryptUI
+//
+//  Created by Ioan Moldoan on 11/16/23
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import AsyncDisplayKit
+
+public final class ThreadDetailWebNode: CellNode {
+    public struct Input {
+        let message: String?
+        let index: Int
+
+        public init(message: String?, index: Int) {
+            self.message = message
+            self.index = index
+        }
+    }
+
+    private let input: ThreadDetailWebNode.Input
+
+    private lazy var webViewNode: CustomWebViewNode = {
+        let node = CustomWebViewNode()
+        node.setAccessibilityIdentifier(accessibilityIdentifier: "aid-message-\(input.index)")
+        node.setHtml("""
+            <header>
+                <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no">
+                <style>
+                    * { font-family: -apple-system, "Helvetica Neue", sans-serif; }
+                    :root { color-scheme: light dark; supported-color-schemes: light dark; }
+                    @media (prefers-color-scheme: dark) {
+                      :root {
+                        background-color: #2D2C2E;
+                        color: white;
+                      }
+                    }
```

Comment:
looks great now, just need to update links color in dark mode as currently they're quite difficult to read:
<img width="200" alt="Screenshot 2023-11-23 at 11 49 42" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/4a4083f7-e237-4c11-b98a-0174086a79d4">

let's use the same color as for links in encrypted messages:
<img width="140" alt="Screenshot 2023-11-23 at 15 13 40" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/5e9a9a72-82a5-4c99-a7b9-6d2aaaf0a107">

Reply:
Sounds good!